website/content/docs/commands/plugin/index.mdx

```diff
@@ -13,35 +13,39 @@ the plugin catalog
 
 ## Examples
 
-List all available plugins in the catalog:
+List all available secret plugins in the catalog:
 
 ```shell-session
-$ vault plugin list
+$ vault plugin list secret
 
-Plugins
--------
-my-custom-plugin
+Name            Version
+----            -------
+ad              v0.14.0+builtin
 # ...
```

Comment:
This "# ..." seems like it should be removed? Looks out of place. Or maybe just the pound sign should be removed while leaving the ellipsis?